src/Attributes.fs

```diff
@@ -0,0 +1,14 @@
+namespace Fabulous.MauiControls.MediaElement
+
+open Fabulous
+open Microsoft.Maui.Controls
+
+module Attributes =
+    
+    let inline defineProperty<'T when 'T: equality> name (defaultValue: 'T) ([<InlineIfLambda>] setter: obj -> 'T -> unit) =
+        Attributes.defineSimpleScalarWithEquality<'T> name (fun _ newValueOpt node ->
+            let target = node.Target :?> BindableObject
```

Comment:
@oco-adam The controller was crashing with InvalidCastingException because we were using `BindableProperty` instead of `BindableObject`. This should fix it :)